deployments/with-creds/ci/values.yaml

```diff
@@ -82,11 +82,20 @@ concourse:
     web:
       auth:
         mainTeam:
+          config: |
+            roles:
+            - name: pipeline-operator
```

Comment:
see comment below - if you decide to take the second approach, you'll want this to be `member`. Otherwise it's fine.

Reply:
If taking second approach, which is demote those two permission to viewer, then the change here should be `viewer` right? 

If taking first approach, then `pipeline-operator` will have only two permissions left, then we don't need to update here.

Just want to confirm.

Reply:
yeah, my bad